ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -1,12 +1,13 @@
 (ns cmr.ingest.api.routes
   "Defines the HTTP URL routes for the ingest API."
+  (:import [java.io File])
   (:require
    [cheshire.core :as json]
    [cmr.acl.core :as acl]
+   [cmr.ingest.data.memory_db]
```

Comment:
why _ and not -

Reply:
Actually I'm going to remove this require and import it instead. But yes, it should have been '-'. I got the imports incorrect

Reply:
Actually, just tried it, and it doesn't work with the '-'. it only worked with the underscore, so keeping as is